data/virt_autotest/virt_logs_collector.sh

```diff
@@ -0,0 +1,427 @@
+#!/bin/bash
+
+#Obtain SLES release version and service pack level
+function get_sles_release() {
```

Comment:
We have similar function in virtlib::get_os_release?

Reply:
I dot not want to source virtlib. I just need this one. And no dependency on virtlib anymore